libflux/src/core/lib.rs

```diff
@@ -1,5 +1,5 @@
 #![cfg_attr(feature = "strict", deny(warnings, missing_docs))]
-#![allow(clippy::unknown_clippy_lints)]
+
```

Comment:
Enable `match_single_binding` clippy linter rule. Unsure why it changed to unknown here.